### PR TITLE
docs: update nerdctl description

### PR DESCRIPTION
**Pull request by imeoer**
Signed-off-by: Yan Song <imeoer@linux.alibaba.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Currently Nydus is supporting the following platforms in container ecosystem:
...
 | Runtime       | [CRI-O / Podman](https://github.com/containers/nydus-storage-plugin)                                            | Run Nydus image with CRI-O or Podman                                                                                                                         | 🚧    |
 | Runtime       | [Docker](https://github.com/dragonflyoss/image-service/tree/master/contrib/docker-nydus-graphdriver)            | Run Nydus image in Docker container with graphdriver plugin                                                                                                  | ✅      |
-| Runtime       | [Nerdctl](https://github.com/containerd/nerdctl)                                                                | Run Nydus image with `nerdctl --snapshotter nydus run ...`                                                                                                   | ✅      |
+| Build/Runtime | [Nerdctl](https://github.com/containerd/nerdctl/blob/master/docs/nydus.md)                                      | Build/Run Nydus image with containerd | ✅      |
```
Comment (changweige) [inline README.md:56]: Is this duplicated with L53? They are both talking using nydus together with containerd

Comment (imeoer) [reply README.md:56]: nerdctl is the containerd client and can build nydus, but snapshotter is the containerd plugin, is it right? How about:

L53: `The containerd remote plugin to run Nydus image`.
L56: `The containerd client to build or run (requires nydus snapshotter) Nydus image`.

Comment (changweige) [reply README.md:56]: Sound good to me

**Review by changweige — APPROVED** [review]
LGTM, thanks

**Pull request merged**